packages/core/src/payment/strategies/stripe-upe/stripe-upe.ts

```diff
@@ -194,7 +194,12 @@ interface ShippingDefaultValues {
 }
 
 interface CustomerDefaultValues {
+    mode?: string;
```

Comment:
This seems like a required property here https://stripe.com/docs/js/elements_object/create_address_element#address_element_create-options-mode ?